### ISSUE TITLE
Error while fetching extensions. XHR failed

### ISSUE THREAD

**Issue by akshaybhor7880** [microsoft/vsmarketplace#214]
Hi I am facing this while installing extensions.
I installed vscode and was not able to see any extension.
when i tried to serach tha following error pooped up below search box.:
.
Error while fetching extensions. XHR failed.
.
Please do tell me how to fix error.
I am using office laptop here.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/75383935/137125631-d9932c12-810c-4ae5-89d9-7ba5feb73fcc.png)

**Comment by sandy081** [comment]
@SaiKanth007 fyi

**Comment by tomthecarrot** [comment]
I'm having the same issue (XHR failed) on Arch Linux after updating to VSCode v1.61.2. My previous installation (which IIRC was v1.59.x) did not have this issue. GitHub account authentication is also not working.

**Comment by niki1312** [comment]
same issue on my windows 10 PC. If anyone has a solution to it please help me...

**Comment by akshaybhor7880** [comment]
I was able to resolve issue on my system.
We had installed security software for browser and it was blocking VScode to download from marketplace.
Just dosabled the software and downloaded the extensions.
Once the security softwares is enabled it again blocked vscode to access chrome for downloads, but installed extensions works fine.

**Comment by prashantvc** [comment]
duplicate https://github.com/microsoft/vsmarketplace/issues/186

**Comment by 24-Fahed** [comment]
By the way, I was using vscode on Ubuntu with GUI. At that time, I turned on both http and socks proxy in the settings application, but I forgot to set the socks port in Clash and then the same error occurred. Later, I set the socks port in the Clash configuration file, and then it was work. Similarly, I don't fill in any settings for socks in the settings application, and only set the http proxy port, it will also work.
